requirements/test.txt

```diff
@@ -11,7 +11,7 @@ astroid==2.11.7
     #   -r requirements/dev.txt
     #   pylint
     #   pylint-celery
-attrs==21.4.0
+attrs==22.1.0
```

Comment:
Only major change is the drop of support for Python2.7
Change-logs: https://www.attrs.org/en/stable/changelog.html#backwards-incompatible-changes